launcher/launcher.go

```diff
@@ -687,46 +687,70 @@ func (launcher *Launcher) getLayersForService(digests []string) ([]imagemanager.
 	return layers, nil
 }
 
-func getStorageRequestRatio(
-	serviceRatios *aostypes.ResourceRatiosInfo, nodeRatios *aostypes.ResourceRatiosInfo,
-) float64 {
-	if serviceRatios != nil && serviceRatios.Storage != nil {
-		return float64(*serviceRatios.Storage) / 100
+func getReqDiskSize(serviceConfig aostypes.ServiceConfig, nodeRatios *aostypes.ResourceRatiosInfo,
+) (stateSize, storageSize uint64) {
+	stateQuota := serviceConfig.Quotas.StateLimit
+	storageQuota := serviceConfig.Quotas.StorageLimit
+
+	if serviceConfig.RequestedResources != nil && serviceConfig.RequestedResources.Storage != nil {
```

Comment:
We should have different values for state and storage. Let's discuss with cloud.

Reply:
fixed